salt/engines/slack.py

```diff
@@ -224,5 +224,6 @@ def fire(tag, msg):
                     else:
                         # Fire event to event bus
                         fire('{0}/{1}'.format(tag, _m['type']), _m)
+            time.sleep(1)
```

Comment:
Checked it again, it was some weird Unicode foobar. Copy-paste related problems. It's fixed in e3a883c... commit.